src/test/regress/expected/multi_modifications.out

```diff
@@ -317,6 +317,11 @@ WITH deleted_orders AS (INSERT INTO limit_orders DEFAULT VALUES RETURNING *)
 UPDATE limit_orders SET symbol = 'GM';
 ERROR:  cannot perform distributed planning for the given modification
 DETAIL:  Common table expressions are not supported in distributed modifications.
+-- updates referencing a column are supported
+UPDATE limit_orders SET bidder_id = bidder_id + 1 WHERE id = 246;
```

Comment:
I think insert is [already](https://github.com/citusdata/citus/blob/240-allow-non-constant-updates/src/test/regress/sql/multi_modifications.sql#L92) pretty [well](https://github.com/citusdata/citus/blob/240-allow-non-constant-updates/src/test/regress/sql/multi_modifications.sql#L115) covered by this test.